sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -258,12 +256,18 @@ public virtual Response<DetectedLanguage> DetectLanguage(string document, string
         /// status code.</exception>
         public virtual async Task<Response<DetectLanguageResultCollection>> DetectLanguageBatchAsync(IEnumerable<string> documents, string countryHint = default, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default)
         {
-            //Argument.AssertNotNullOrEmpty(documents, nameof(documents));
-            //options ??= new TextAnalyticsRequestOptions();
-            //LanguageDetectionAnalysisInput detectLanguageInputs = ConvertToLanguageInputs(documents, countryHint);
-            //return await DetectLanguageBatchAsync(detectLanguageInputs, options, cancellationToken).ConfigureAwait(false);
-            await Task.Yield();
-            throw new NotImplementedException();
+            Argument.AssertNotNullOrEmpty(documents, nameof(documents));
+            options ??= new TextAnalyticsRequestOptions();
```

Comment:
Since we tend to use this pattern a bunch and I'm assuming that we're treating options immutably - should we conisder creating a `static readonly`  default instance of the options and then using them rather than allocating per-call?    Something like:

```csharp
public class TextAnalyticsClient
{
    private static readonly TextAnalyticsRequestOptions _defaultRequestOptions = new();

    // ... CODE HERE

    public virtual async Task<Response<DetectLanguageResultCollection>> DetectLanguageBatchAsync(IEnumerable<string> documents, string countryHint = default, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default)
    {
          Argument.AssertNotNullOrEmpty(documents, nameof(documents));
          options ??= _defaultRequestOptions;

         // ... MORE CODE
    }
}
```